lib/battle/ReachabilityInfo.h

```diff
@@ -27,7 +27,7 @@ struct DLL_LINKAGE ReachabilityInfo
 	{
 		const CStack * stack; //stack for which calculation is mage => not required (kept for debugging mostly), following variables are enough
 
-		bool attackerOwned;
+		ui8 side;

```

Comment:
Might be we should use strongly typed ENUM instead or something?
Since replacing true/false with 0/1 doesn't make it easier to understand.

I suppose H3 always put attacker on left side and defenders on the right so...
Though I wish to know what @FeniksFire thinking about it since he was working on battle code refactoring.